azure-pipelines.yml

```diff
@@ -22,28 +22,4 @@ steps:
     testRunner: VSTest
     testResultsFiles: '**/*.trx'
 
-#- task: PowerShell@2
-#  displayName: Run ProcessCodeCoverage.ps1
-#  inputs:
-#    targetType: filePath
-#    filePath: ./scripts/ProcessCodeCoverage.ps1
-
-#- task: PowerShell@2
-#  displayName: 'Merging TestResults'
-#  inputs:
-#    targetType: 'inline'
-#    script: |
-#      dotnet tool install -g dotnet-reportgenerator-globaltool      
-#      reportgenerator -reports:**/*.coverage.xml -targetdir:TestResults -reporttypes:Cobertura
-
-#- task: PublishCodeCoverageResults@1
-#  inputs:
-#    codeCoverageTool: 'cobertura'
-#    summaryFileLocation: 'TestResults/Cobertura.xml'
-
-# Enable if you need to debug the tests
-#- task: PublishPipelineArtifact@1
-#  condition: always()
-#  inputs:
-#    targetPath: bld/bin/AnyCPU_Debug/Tests.Security/netcoreapp3.1/
-#    artifactName: Tests.Security
+- task: ComponentGovernanceComponentDetection@0
```

Comment:
>ComponentGovernanceComponentDetection@0 [](http://example.com/codeflow?start=8&length=39)

this will guarantee that ComponentGovernance can always track this repository